README.md

```diff
@@ -4,5 +4,5 @@ Commit 1<br>
 Commit 2<br>
 Commit 3
 
-I've been looking for **pizza** for several hours.
+I've been looking for **pizza** for several hours. And i was waiting too long
```

Comment:
You must add <br> after word "hours", so you write it down on the new line. thanks

Reply:
Hello, thanks for reviewing my code, i've changed my code as you suggested.

Reply:
Thanks for contributing on this repo, you did the great work today. I appreciate it.